language/move-vm/runtime/src/loader.rs

```diff
@@ -2155,13 +2170,15 @@ struct FieldInstantiation {
 struct StructInfo {
     struct_tag: Option<StructTag>,
     struct_layout: Option<MoveStructLayout>,
+    annotated_struct_layout: Option<MoveStructLayout>,
 }
 
 impl StructInfo {
     fn new() -> Self {
         Self {
             struct_tag: None,
             struct_layout: None,
+            annotated_struct_layout: None,
```

Comment:
I'm not super thrilled by this new field, but I think #525 might make the impact a lot smaller